### ISSUE TITLE
Support for Red-Hat based systems

### ISSUE THREAD

**Issue by jtagcat** [vinifmor/bauh#36]
Make it the universal package manager then ;)

**Comment by vinifmor** [comment]
Hi @jtagcat, do you mean a bauh package for installing in Red-hat based systems or to handle such format ?

**Comment by jtagcat** [comment]
A package to install via yum/dnf.
Compatability with packages from yum/dnf sources would also be commended!

**Comment by vinifmor** [comment]
I will add to the backlog

**Comment by AWHubGit** [comment]
Red Hat will switch to atomic updates with Fedora Silverblue. Completely different technique to do updates.
https://fedoramagazine.org/fedora-silverblue-brings-future-tech-to-the-desktop/

**Comment by jtagcat** [comment]
Fair point, I haven't made it to Silverblue yet, a better title would be this:

**Comment by vinifmor** [comment]
I've already tried to port bauh as a Flatpak, but Flatpak has some permission issues that I couldn't pass through. Bauh ends up not working properly, since it needs to call a lot of host's installed programs. I'm going to open a PullRequest with the tag **needs help** if someone wants to have a look and possibly fix the permission issues.

**Comment by vinifmor** [comment]
[#115](https://github.com/vinifmor/bauh/issues/115)
